src/couch_views/src/couch_views_indexer.erl

```diff
@@ -368,6 +382,7 @@ write_docs(TxDb, Mrst, Docs, State) ->
     if LastSeq == false -> ok; true ->
         couch_views_fdb:set_update_seq(TxDb, Sig, LastSeq)
     end,
+    update_tasks(length(Docs), LastSeq),
```

Comment:
I think you can use `DocsNumber` since that is the count of the number of docs processed